### DISCUSSION TITLE
[bug]: [/lnrpc.Lightning/GetInfo]: expected 1 macaroon, got 0

### DISCUSSION THREAD

**Discussion by Impa10r** [lightningnetwork/lnd#9297] [Troubleshooting]
### Background

My testnet VM hang and had to be powered down. On restart LND fills the log with this message every second and is not responsive. I already tried wiping .macaroon and macaroons.db files. I also tried uninstalling and reinstalling Lightning Node app, with copying the wallet.db and channels.db files. The issue persists.

I don't know who is calling rpc getinfo. Only Bitcoin Core and LND are running on this Umbrel. 

### Your environment

* version of `lnd` 0.18.3
* which operating system (`uname -a` on *Nix) Debian 11
* version of `btcd`, `bitcoind`, or other backend bitcoind
* any other relevant environment details Umbrel 1.0

**Comment by guggero** [comment]
Are you sure you're not running Lightning Terminal or Lightning Shell on your Umbrel?
The log message means someone or something calls `GetInfo` without a macaroon. That's not an `lnd` issue but something on your system.

**Reply by Impa10r** [reply]
It must be Lightning Node frontend app:

```
root@debian:/home/umbrel# docker logs lightning_app_1 --follow
Checking LND status...
Waiting for LND...
Checking LND status...
Waiting for LND...
Checking LND status...
Waiting for LND...
Checking LND status...
```

**Comment by Impa10r** [comment]
Found the cause: https://github.com/getumbrel/umbrel-apps/issues/1885
